### DISCUSSION TITLE
How to recognize ancient Chinese characters on bamboo slips?

### DISCUSSION THREAD

**Discussion by PeterH0323** [PaddlePaddle/PaddleOCR#6021] [Q&A]
When they are common Chinese characters, just annotate the adequate amount of data and fine-tune models. If there is not enough data, you can try StyleText. But when they are special characters like inscriptions on oracle and hieroglyphs, it is necessary to make a specialized dictionary and train models.

**Comment by xiaoyangyang2** [comment]
Prepare a dictionary containing all the characters of the working scene and label it with enough rich data and then try to train
